packages/next/cli/next-lint.ts

```diff
@@ -18,7 +18,16 @@ import { getProjectDir } from '../lib/get-project-dir'
 
 const eslintOptions = (args: arg.Spec, defaultCacheLocation: string) => ({
   overrideConfigFile: args['--config'] || null,
-  extensions: args['--ext'] ?? ['.js', '.jsx', '.ts', '.tsx'],
+  extensions: args['--ext'] ?? [
+    '.js',
+    '.jsx',
+    '.cjs',
+    '.mjs',
+    '.ts',
+    '.tsx',
+    '.cts',
+    '.mts',
```

Comment:
Are these extensions being handled by the default parser already? It looks like they may need to be added here as well https://github.com/vercel/next.js/blob/e9659113f0e13f985c523ec140cc6d979ab41e07/packages/eslint-config-next/index.js#L109

Would be good to add a test for these as well.

Reply:
Has this been taken into consideration?